src/js/element.js

```diff
@@ -0,0 +1,133 @@
+import { ELEMENT_COLORS } from "./element_colors";
+import { PERIODIC_TABLE } from "./periodic_table.js";
+import { CONVERSION, UNITS, convertLength, convertEnergy } from "./units";
+
+export class Element {
+    constructor(symbol) {
+        this.symbol = symbol;
+        this._properties = PERIODIC_TABLE[this.symbol];
+    }
+
+    get name() {
+        return this._properties.name;
+    }
+
+    get symbol() {
+        return this._symbol;
+    }
+
+    set symbol(symbol) {
+        if (Element.isValidSymbol(symbol, true)) {
+            this._symbol = symbol;
+        } else if (Element.isValidSymbol(symbol, false)) {
+            this._symbol = symbol.charAt(0).toUpperCase() + symbol.slice(1).toLowerCase();
+        } else {
+            throw new Error(`Invalid symbol: ${symbol}`);
+        }
+    }
+
+    /**
+     * @summary Returns atomic mass in amu.
+     * @returns {number|undefined} - Atomic mass in amu
+     */
+    get mass() {
+        if (Number.isFinite(this._properties.atomic_mass)) {
```

Comment:
1. The pattern is a bit repetitive - maybe we should define a checker function
2. `typeof` would probably be a more clear check

Reply:
I used `typeof` instead. For the checker function, perhaps we can validate using schemas here?